lib/features/Auth/signup/presentation/widgets/password_validation_rules.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mentorship_e1_g3/features/Auth/signup/presentation/cubit/signup_cubit.dart';
+
+import 'validator_text.dart';
+
+class PasswordValidationRules extends StatelessWidget {
+  const PasswordValidationRules({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    SignupCubit cubit = BlocProvider.of<SignupCubit>(context);
+    return BlocBuilder<SignupCubit, SignupState>(
```

Comment:
why BlocBuilder here ?

Reply:
because the password field wouldn't rebuild until i put this BlocBuilder , but i notice that issue because the Const keyword , so will remove the const then will remove the this Blocbuilder too